### ISSUE TITLE
Enable UDP port 21324 in trezor-user-env-mac

### ISSUE THREAD

**Issue by matejkriz** [trezor/trezor-user-env#55]
To be able to use Trezor emulator with Suite in Android simulator, we need to be able to communicate on UDP port 21324. 

Adding 
```yml
- "21324:21324/udp"
```
in compose.yml doesn't work for me.

Testing using this node script:
```javascript
var udp = require("dgram");

function callback(msg, info) {
  console.log("Data received from server : " + msg.toString());
  console.log(
    "Received %d bytes from %s:%d\n",
    msg.length,
    info.address,
    info.port
  );
}

var client = udp.createSocket("udp4", callback);

var data = Buffer.from("PINGPING");

client.on("message", callback);

client.send(data, 21324, "127.0.0.1", function (error) {
  if (error) {
    client.close();
  } else {
    console.log("Data sent !!!");
  }
});
```

It does work on linux using `network_mode: "host"`, at least for @szymonlesisz 

And it works fine if I run stadalone Trezor emulator outside Docker, so this it the way to develop Android app in simulator so far.

**Comment by tsusanka** [comment]
> To be able to use Trezor emulator with Suite in Android simulator, we need to be able to communicate on UDP port 21324.

This is true for Android because it obviously does not use [Bridge](https://github.com/trezor/trezord-go) but some React Native thing right? For desktop we do not have to expose emulator because we communicate with it via Bridge. Am I right? Just putting my thoughts together :)

**Comment by matejkriz** [comment]
@tsusanka Yes, React Native has its own bridge. [The UDP one](https://github.com/trezor/trezor-suite/blob/develop/packages/suite-native/android/app/src/main/java/io/trezor/suite/bridge/UDPBridge.java) is there for development purposes only because it's not possible to connect USB device to Android simulator.

**Comment by matejkriz** [comment]
yeah, I have put it in `trezor-user-env-mac` section and even trying to run it without compose using `docker run` with the same params

**Comment by tsusanka** [comment]
That is curious. @vdovhanych any idea by any chance? :)

**Comment by vdovhanych** [comment]
Unfortunately, I cannot test it on my Mac as it's M1. I cant run emulators yet (working on it). Tested it on Linux, and yes, it does work there. As to the problem with mac, I don't think it's related to the ports. It opens them as supposed to, but Matej's script isn't getting the response from the emulator.

**Comment by tsusanka** [comment]
@matejkriz could you share what your script returns when run in the docker setup? I suck at reading JS

**Comment by matejkriz** [comment]
@tsusanka the testing script returns only `Data sent !!!` when the emulator run in docker.

When I run the standalone emulator and bridge built on my Mac, I get:
```
Data sent !!!
Data received from server : PONGPONG
Received 8 bytes from 127.0.0.1:21326

Data received from server : PONGPONG
Received 8 bytes from 127.0.0.1:21326
```

**Comment by tsusanka** [comment]
One more thing to clarify: @matejkriz do you see the emulator? I mean does it launch? So you see it in your OS but the communication does not work_

**Comment by matejkriz** [comment]
yes, the emulator does launch, just no response on PINGPING message

**Comment by matejkriz** [comment]
will try

cc @Nodonisko
